src/Controller/ModulesController.php

```diff
@@ -477,6 +481,39 @@ public function relationshipsJson($id, string $relation) : void
         $this->set('_serialize', array_keys($response));
     }
 
+    /**
+     * Retrieve thumbnails URL of related objects in `meta.url` if present.
+     *
+     * @param array $response Related objects response.
+     * @return void
+     */
+    public function getThumbsUrls(array &$response) : void
+    {
+        if (empty($response['data'])) {
+            return;
+        }
+
+        // extract ids of objects
+        $ids = Hash::extract($response, 'data.{n}.id');
+        if (empty($ids)) {
+            return;
+        }
+
+        $thumbs = '/media/thumbs?ids=' . implode(',', $ids) . '&options[w]=400';
```

Comment:
* do we try to create thumbs for all object types or only for media types? 
* dont' particularly like the hardcoded `=400`, maybe define a constant in a view or helper class?